recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
```

Comment:
Well, I'm sorry to here this, but I'm inclined not to merge this without a change here. As they mention in their own documentation, this package contains so much software that "...each Latest release is bound to have a bug of major impact in one of them". Selecting a very stable tarball provides usable desirable software. Selecting the latest version that almost certainly has a bug of major impact is not.